QUARKUS-2492.md

```diff
@@ -13,11 +13,13 @@ Test development will focus on
 - Quarkus JDBC transactions based on programmatic approach
 - Quarkus JDBC transactions based on lambda approach
 - Verify commons transactions states as "begin", "commit" and "rollback"
-- Verify classic and reactive flavors
+- Reactive flavor is out of scope, currently only classic approach is supported
 - The following databased will be covered: Oracle, Mysql, MariaDB, Mssql, Postgresql
+- In order to ensure a good integration with other key extensions that are essential for a production ready microservice 
```

Comment:
Is this needed? What's the connection between transactions and OpenAPI, Tracing and Metrics?

You want to rollback transaction and see how it's reflected in metrics?

I can't think about the checks around OpenAPI and Tracing  with Transaction API. Please share your ideas.

Reply:
I'm not sure if we need that custom metric check in our TS, but it can stay there if you want ... But pls adjust the text in the PR to mention just Metric and explicitly say you are using custom metric handled by the application code.

Reply:
Pls use this text, it's covering all the topics and actions performed around this Tp
```
- Areas like OpenAPI, Tracing and Metrics were considered when preparing the test plan
  - Closer integration for Tracing and Metrics topic will be evaluated and brought to the development team
  - TD will cover custom metric handled by the application code and ensure true transactional behaviour - e.g. rollback
```